src/backend/commands/player/kick/multi_kick.cpp

```diff
@@ -18,12 +18,12 @@ namespace big
 		virtual void execute(player_ptr player, const command_arguments& _args, const std::shared_ptr<command_context> ctx) override
 		{
 			if (g_player_service->get_self()->is_host())
-				dynamic_cast<player_command*>(command::get("breakup"_J))->call(player, {}),
-				    dynamic_cast<player_command*>(command::get("hostkick"_J))->call(player, {});
-
-			dynamic_cast<player_command*>(command::get("oomkick"_J))->call(player, {});
-			script::get_current()->yield(700ms);
+				dynamic_cast<player_command*>(command::get("breakup"_J))->call(player, {});
 
+			if (player->is_host()) {
+				dynamic_cast<player_command*>(command::get("oomkick"_J))->call(player, {});
+				script::get_current()->yield(700ms);
+			}
 
 			if (player && !g_player_service->get_self()->is_host() && player->is_valid() && !player->is_host())
```

Comment:
I'd recommend re-ordering the checks so they're a bit easier to follow.
First check that we're not the host, then check that player is not null, then if they're valid, then if they're not host.

if (!g_player_service->get_self()->is_host() && player && player->is_valid() && !player->is_host())